src/component/Modal.tsx

```diff
@@ -200,10 +202,17 @@ class Modal extends Component<Props, State> {
         if (scrollableElem) {
             (scrollableElem as HTMLElement).focus();
         }
+
+        if (this.props.focusVendors) {
+            const vendorsElem = window.document.getElementById(VENDORS_ID);
+            if (vendorsElem) {
+                vendorsElem.scrollIntoView({ behavior: 'smooth' });
```

Comment:
Note: Setting the behaviour is currently only supported by Edge, Firefox, Chrome and Opera but polyfilling it means bloating our bundle. The behaviour on others browsers is that the window snaps to the Vendors. Seeing as only a small number of users surface the modal and this CMP will soon be replaced by Sourcepoint's I opted not to polyfill it.

Reply:
nice!